core/variant/variant_call.cpp

```diff
@@ -2392,6 +2393,7 @@ static void _register_variant_builtin_methods_array() {
 	bind_method(Array, bsearch_custom, sarray("value", "func", "before"), varray(true));
 	bind_method(Array, reverse, sarray(), varray());
 	bind_method(Array, duplicate, sarray("deep"), varray(false));
+	bind_method(Array, duplicate_deep, sarray("deep_subresources_mode"), varray(RESOURCE_DEEP_DUPLICATE_NONE));
```

Comment:
Shouldn't this also use RESOURCE_DEEP_DUPLICATE_INTERNAL?

Reply:
Good catch. Fixed in the upcoming push.